test/integration/test-errors.js

```diff
@@ -52,7 +52,7 @@ t.run('error page', function() {
   });
 
   it.configure().skipFirefox().skipEdge()
-      .run('should show the body in error test', () => {
+      .skip('should show the body in error test', () => {
```

Comment:
All `skip`s should ideally have a TODO, but let's not block this PR purely for this.